daemon/monitor_windows.go

```diff
@@ -16,9 +16,19 @@ func platformConstructExitStatus(e libcontainerd.StateInfo) *container.ExitStatu
 
 // postRunProcessing perfoms any processing needed on the container after it has stopped.
 func (daemon *Daemon) postRunProcessing(container *container.Container, e libcontainerd.StateInfo) error {
-	//TODO Windows - handle update processing here...
 	if e.UpdatePending {
-		return fmt.Errorf("Windows: Update handling not implemented.")
+		spec, err := daemon.createSpec(container)
+		if err != nil {
+			return err
+		}
+
+		spec.Windows.Servicing = true
```

Comment:
Overall, I'm generally OK with this PR, except for this specific piece. Rather than have the servicing flag in the spec for the container, I think we should have this as an option on the containerd.Create call.